e2e/utils/functions/openmrs.ts

```diff
@@ -284,13 +286,16 @@ export class OpenMRS {
     await this.page.getByLabel('Quantity to dispense').fill('12');
     await this.page.getByLabel('Prescription refills').fill('3');
     await this.page.getByPlaceholder('e.g. "Hypertension"').type('Hypertension');
+  }
+
+  async saveDrugOrder() {
     await this.page.getByRole('button', { name: 'Save order' }).focus();
     await expect(this.page.getByText('Save order')).toBeVisible();
     await this.page.getByRole('button', { name: 'Save order' }).click();
     await this.page.getByRole('button', { name: 'Sign and close' }).focus();
     await expect(this.page.getByText('Sign and close')).toBeVisible();
     await this.page.getByRole('button', { name: 'Sign and close' }).click();
-    await delay(5000);
+    await delay(3000);
```

Comment:
Why the decrease?

Reply:
It takes less than 3 seconds to have the drug order saved.